### PR TITLE
Change GitHub Organizations

### PR DESCRIPTION
**Pull request by DLu**
I tried to follow the link on the bottom of [this page](https://docs.nav2.org/tuning/index.html#other-pages-we-d-love-to-offer) and hit a 404. I programmatically replaced the following strings:
 * `ros-planning/navigation2` :arrow_right: `ros-navigation/navigation2`
 * `ros-planning/docs` :arrow_right: `ros-navigation/docs`

This follows the organizational changes mentioned [here](https://discourse.ros.org/t/osrf-policy-on-github-organisation-naming/37642) and [here](https://discourse.ros.org/t/the-new-home-of-ros-navigation-ros-navigation/37657)

### CODE REVIEW

File: substitutions.txt [hunk 1]
```diff
@@ -5,19 +5,19 @@
...
-.. _documentation repo: https://github.com/ros-planning/docs.nav2.org/doc
+.. _documentation repo: https://github.com/ros-navigation/docs.nav2.org/doc
 
-.. _nav2_bringup pkg: https://github.com/ros-planning/navigation2/tree/main/nav2_bringup/bringup
+.. _nav2_bringup pkg: https://github.com/ros-navigation/navigation2/tree/main/nav2_bringup/bringup
 
-.. _nav2_params file: https://github.com/ros-planning/navigation2/blob/main/nav2_bringup/bringup/params/nav2_params.yaml
+.. _nav2_params file: https://github.com/ros-navigation/navigation2/blob/main/nav2_bringup/bringup/params/nav2_params.yaml
 
 .. _documentation site: https://rosplanning.github.io/docs.nav2.org/
```
Comment (DLu) [inline substitutions.txt:16]: Not sure if these need changes too?

**Review by SteveMacenski — APPROVED** [review]
tak (thank you in Danish - see you in Europe)!

File: development_guides/build_docs/index.rst [hunk 2]
```diff
@@ -121,14 +121,14 @@ Once your system is setup, you can build the Nav2 Dockerfile from the root of th
 .. code:: bash
 
   export ROS_DISTRO=rolling
-  git clone https://github.com/ros-planning/navigation2.git --branch main
+  git clone https://github.com/ros-navigation/navigation2.git --branch main
   docker build --tag navigation2:$ROS_DISTRO \
     --build-arg FROM_IMAGE=ros:$ROS_DISTRO \
     --build-arg OVERLAY_MIXINS="release ccache lld" \
-    --cache-from ghcr.io/ros-planning/navigation2:main \
+    --cache-from ghcr.io/ros-navigation/navigation2:main \
```
Comment (DLu) [inline development_guides/build_docs/index.rst:128]: This is the only instance where it wasn't pointing at github.com

File: substitutions.txt (hunk 1 above)
Comment (SteveMacenski) [reply substitutions.txt:16]: Oh - yes probably!

File: development_guides/build_docs/index.rst (hunk 2 above)
Comment (SteveMacenski) [reply development_guides/build_docs/index.rst:128]: This is a correct update

File: substitutions.txt (hunk 1 above)
Comment (DLu) [reply substitutions.txt:16]: Can you check that Github pages is enabled? Neither the existing link or https://rosnavigation.github.io/docs.nav2.org/ work at the moment.

Comment (SteveMacenski) [reply substitutions.txt:16]: It should just be docs.nav2.org. The GitHub pages is redirected to that domain in its configuration

**Review by SteveMacenski — APPROVED** [review]

**Comment by SteveMacenski** [comment]
Thanks @DLu !

File: plugins/index.rst [hunk 3]
```diff
@@ -96,52 +96,47 @@ Costmap Filters
...
-| `Rotation Shim Controller`_    | Steve Macenski        | A "shim" controller to rotate      | Differential, Omni,   |
-|                                |                       | to path heading before passing     | model rotate in place |
-|                                |                       | to main controller for  tracking.  |                       |
-+--------------------------------+-----------------------+------------------------------------+-----------------------+
-| `Graceful Controller`_         | Alberto Tudela        | A controller based on a            | Differential          |
-|                                |                       | pose-following control law to      |                       |
-|                                |                       | generate smooth trajectories.      |                       |
-+--------------------------------+-----------------------+------------------------------------+-----------------------+
-| `Vector Pursuit Controller`_   | Black Coffee Robotics | A controller based on the vector   | Differential,         |
```
Comment (exMachina316) [inline plugins/index.rst:133]: Vector pursuit was removed from the list NAV2 supported controllers. Can we please revert this change back? @SteveMacenski @DLu

Comment (SteveMacenski) [reply plugins/index.rst:133]: @exMachina316 im on the go still traveling post-ROSCon, can you submit a PR to readd, that would be there most straight forward I think.

Comment (DLu) [reply plugins/index.rst:133]: #602

Comment (exMachina316) [reply plugins/index.rst:133]: Thanks @DLu!!
